composer.json

```diff
@@ -22,11 +22,20 @@
         "address",
         "addresses",
         "subnet",
-        "matching"
+        "matching",
+        "managing",
+        "manage"
     ],
     "require": {
         "php": ">=5.3.3"
     },
+    "require-dev": {
+        "ext-json": "*",
+        "ext-pdo": "*",
+        "ext-pdo_sqlite": "*",
```

Comment:
As I already wrote, adding dependencies to `require-dev` is not so harmless. When I added it originally I wasn't aware of that. So I now prefer to be very picky about it.